DashboardSettings: Fixes issue with tags list not updating changes are made (#32189)
grafana/grafana: docs/sources/datasources/cloudwatch.md

Please see the AWS documentation for [Service Quotas](https://docs.aws.amazon.com/servicequotas/latest/userguide/intro.html) and [CloudWatch limits](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_limits.html) for more information.

## Configure the data source with grafana.ini

> In the [Grafana configuration](https://grafana.com/docs/grafana/latest/administration/configuration/#aws) there's an `AWS` section that allows you to customize the data source.

Reviewer: ```suggestion
In the [Grafana configuration](https://grafana.com/docs/grafana/latest/administration/configuration/#aws) file, the `AWS` section allows you to customize the data source.
```
